lib/routes/account.js

```diff
@@ -1087,11 +1087,18 @@ module.exports = function (
             }
           )
           .then(
-            function (accountRecord) {
+            function (account) {
               log.activityEvent('account.reset', request, {
-                uid: accountResetToken.uid.toString('hex')
+                uid: account.uid.toString('hex')
               })
-              return customs.reset(accountRecord.email)
+              log.event(
+                'reset',
+                {
+                  uid: account.uid.toString('hex') + '@' + config.domain,
```

Comment:
I just noticed that some `log.event` calls use `{ uid: uid+domain }` and others use `{ uid: uid }`, and then I realized that it depends entirely on whether they're to be consumed by tokenserver or not, and then I died a little inside...